src/chess/piece/King.java

```diff
@@ -19,6 +19,17 @@ public boolean isValidMove(Position newPosition) {
       return false;
     }
     // ToDo: implement
+    // if difference of Col & Row is 1, it can move
+    Position currentPosition = null;
+    int diffOfCol = Math.abs(currentPosition.getCol() - newPosition.getCol());
+    int diffOfRow = Math.abs(currentPosition.getRow() - newPosition.getRow());
```

Comment:
What do you mean by line 23? `Position currentPosition = null;`
You're using this variable on next line by accessing the `getCol` method of it, so it'll be NullPointerException
Please check how it works and fix it up